modules/opm-cli-ref-generate.adoc

```diff
@@ -63,12 +57,15 @@ $ opm generate dockerfile <dcRootDir> [flags]
 |Flag |Description
 
 |`-i,` `--binary-image` (string)
-|Image in which to build catalog. The default value is `quay.io/operator-framework/opm:latest`.
+|Base image with which to build the catalog. The default value is the upstream `quay.io/operator-framework/opm:latest` image.
+ifndef::openshift-origin[]
+To build with the official Red Hat image, use `registry.redhat.io/openshift4/ose-operator-registry:v{product-version}`.
```

Comment:
`v{product-version}` doesn't render correctly in the netlify version, showing as 'vBranch Build'.

Reply:
Same re: above wrt working branch.